content/en/apps/examples/covid-rdt-reference-app.md

```diff
@@ -0,0 +1,521 @@
+---
+title: "Reference Application using RDT for COVID-19 Testing"
+linkTitle: "COVID-19 Testing"
+weight: 
+description: >
+ A customizable CHT application with support for 3rd party RDT integration   
```

Comment:
Revisiting to align with titles and descriptions in [index page](https://docs.communityhealthtoolkit.org/apps/examples/).

```suggestion
title: "COVID-19 Testing with Rapid Diagnostic Tests"
linkTitle: "COVID-19 Testing"
weight: 
description: >
 CHT example application that uses a third party app to capture the result of a Rapid Diagnostic Test.
```